payloads/assignpublicIP.go

```diff
@@ -26,37 +26,43 @@ type PublicIPCommand struct {
 	VnicMAC          string `yaml:"vnic_mac"`
 }
 
-// CommandAssignPublicIP is reserved for future use.
+// CommandAssignPublicIP wrapper of PublicIPCommand used during assigment of IP.
 type CommandAssignPublicIP struct {
 	AssignIP PublicIPCommand `yaml:"assign_public_ip"`
 }
 
-// CommandReleasePublicIP is reserved for future use.
+// CommandReleasePublicIP wrapper of PublicIPCommand used during release of IP.
 type CommandReleasePublicIP struct {
 	ReleaseIP PublicIPCommand `yaml:"release_public_ip"`
 }
 
-// PublicIPFailureReason is reserved for future use.
+// PublicIPFailureReason data type to define reasons of failure during Public IP
```

Comment:
`PublicIPFailureReason represents the potential AssignPublicIP/ReleasePublicIP commands failure reasons.`